src/components/atoms/AboutText.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+	<p class="about__text">{{about.text}}</p>
+</template>
+
+<script>
+	import json from "../../../data/structure.json"
+	export default {
+		name: "aboutText",
+		data() {
+			return {
+				about: json.about
+			}
+		}
+	}
+</script>
+
+<style scoped>
```

Comment:
style lang isn't specified

Reply:
Yes, I will add it, already have loader for it